WcaOnRails/app/webpacker/components/StaticPages/TeamsCommittees.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button, Icon, Popup } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import UserBadge, { subtextForMember, subtextForOfficer } from '../UserBadge';
+import '../../stylesheets/static_pages/teams_committees.scss';
+
+function TeamsCommittees({ officers = [], teams = [], officerTitles = [] }) {
+  return (
+    <>
+      <h1>{I18n.t('about.structure.teams_committees_councils')}</h1>
+      <p>
+        {I18n.t('about.structure.committees')}
+      </p>
+
+      <h3 id="officers">{I18n.t('about.structure.officers.name')}</h3>
+      <p>{I18n.t('about.structure.officers.description')}</p>
+
+      <div className="team-members">
+        {officers.map((user) => (
+          <div key={(user.wca_id || 'user') + user.id}>
+            <UserBadge
+              user={user}
+              badgeClasses="board"
+              subtexts={subtextForOfficer(user, officerTitles)}
+            />
+          </div>
+        ))}
+      </div>
+
+      {teams.map((team) => (
+        <div className="team" key={team.id}>
+          <h3>
+            <span className="name">{team.name}</span>
+            {team.acronym && team.acronym !== 'BOARD' && (
+              <span className="acronym">
+                {team.acronym ? `(${team.acronym})` : ''}
```

Comment:
Won't we already know that `team.acronym` is truthy inside the span?

Reply:
Yep, not sure why I've done that :P